src/jswrap_interactive.c

```diff
@@ -507,7 +508,10 @@ void _jswrap_interface_clearTimeoutOrInterval(JsVar *idVar, bool isTimeout) {
       jsvRemoveChild(timerArrayPtr, child);
       jsvUnLock2(child, timerArrayPtr);
     } else {
-      jsExceptionHere(JSET_ERROR, isTimeout ? "Unknown Timeout" : "Unknown Interval");
+      if (isTimeout)
+        jsExceptionHere(JSET_ERROR, "Unknown Timeout");
```

Comment:
Again, just personal code style? I assume the original option probably uses marginally less flash memory